tables/TaQL/ExprNodeRep.h

```diff
@@ -52,7 +52,7 @@ template<class T> class Block;
 
 //# Define a shared pointer to the Rep class.
 class TableExprNodeRep;
-typedef CountedPtr<TableExprNodeRep> TENShPtr;
+typedef std::shared_ptr<TableExprNodeRep> TENShPtr;
```

Comment:
Optional: Within team Schaap we are moving away from typedefs for std::shared_ptr's, since they have little value and make the code harder to read. Since you're updating code that uses `TENShPtr` anyway, please consider removing this typedef. When using `std::make_shared`, you don't even need it.

Reply:
It is used in many places. I think it makes the code easier to read because it (IMHO) makes it easier to grasp a statement.